FreeRTOS/FreeRTOS-Plus-TCP
# Fix DNS Callback - DNS Networking UT for coverage

Fix DNS Callback - DNS Networking UT for coverage

Description
-----------
Add UT for IPv6 branch for 100% coverage 

Test Steps
-----------
Normal Unit Tests compilation method

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
